pbtools/c_source/__init__.py

```diff
@@ -1307,17 +1334,18 @@ def generate(self):
         return header, source
 
 
-def generate(namespace, parsed, header_name):
+def generate(namespace, parsed, header_name, gen_opts):
     """Generate C source code from given parsed proto-file.
 
     """
 
-    return Generator(namespace, parsed, header_name).generate()
+    return Generator(namespace, parsed, header_name, gen_opts).generate()
 
 
 def generate_files(infiles,
                    import_paths=None,
-                   output_directory='.'):
+                   output_directory='.',
+                   gen_opts=None):
```

Comment:
I'm fine with naming it `options` instead of `gen_opts`.

Reply:
change have been made